### ISSUE TITLE
Timeline avatar contrast fails AA

### ISSUE THREAD

**Issue by Stanton** [jadu/pulsar#1237]
**Describe the bug**
Timeline avatar contrast fails AA

![Screenshot 2020-05-26 at 15 37 20](https://user-images.githubusercontent.com/18653/82916529-0ce67100-9f6a-11ea-910c-d0557528d490.png)

**Comment by jamesjacobs** [comment]
I'm sure this has been fixed in develop.
